octobot_backtesting/api/exchange_data_collector.py

```diff
@@ -36,6 +36,10 @@ async def initialize_and_run_data_collector(data_collector):
     return data_collector.file_name
 
 
+async def stop_data_collector(data_collector):
+    return await data_collector.stop(should_stop_database=False) if data_collector else False
```

Comment:
👍 
If we add kwargs, we should add a **kwargs in `data_collector.py # async def stop(self) -> None:`

Reply:
ah, yes !